.circleci/config.yml

```diff
@@ -204,21 +204,19 @@ jobs:
             - restore_cache:
                 name: Restore cache - opam
                 keys:
-                    - opam-v7-{{ checksum "opam_ci_cache.sig" }}
-                    - opam-v7-igyglTvPBxMMa2HOSyL8o7xJusIN4L6cuLaccWdS7pY=
+                    - opam-v8-{{ checksum "opam_ci_cache.sig" }}
             - run:
                 name: Ensure/Fix opam permisssions
                 command: |
                     set +e
-                    sudo mkdir -p /Users/distiller/.opam
-                    sudo chown -R distiller /Users/distiller/.opam | true
-                    sudo chmod -R u+w /Users/distiller/.opam | true
+                    ([[ -d /Users/distiller/.opam ]] && (sudo chown -R distiller /Users/distiller/.opam | true)) || true
```

Comment:
Can we not fix these before they go into the cache? Does the cache not store enough unix metadata?